cmd/limactl/start.go

```diff
@@ -510,6 +514,15 @@ func startAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if runtime.GOOS != "windows" {
+		foreground, err := cmd.Flags().GetBool("foreground")
+		if err != nil {
+			return err
+		}
+		if foreground {
+			ctx = start.WithLaunchHostAgentForeground(ctx, foreground)
```

Comment:
Is there any reason to use a context for this?
Wondering if we can just add an argument to `start.Start()`

Reply:
There's no particular reason. I just mimicked the way the timeout flag is written.

Reply:
The timeout is contextualized because it has to be passed to many functions.
This is different from the foreground flag that is only used once.

Reply:
Understood, I'll try changing it.